lib/utils.py

```diff
@@ -10,6 +13,10 @@
 ROOT = DIST / 'alerts'
 
 
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
```

Comment:
As discussed in the call, this is basically the filtering of log messages. Anything lower than `INFO` will be ignored [[docs](https://docs.python.org/3/library/logging.html#logging.Logger.setLevel)]

Reply:
That's interesting: "the root logger is created with level WARNING". It's nice Python applies a sensible default, but also surprising since we need to deliberately un-filter. Thanks for helping me learn 😄.

Reply:
Fixed in https://github.com/alphagov/notifications-govuk-alerts/pull/163/commits/dabf64b3f868e86b52e6196c6f65797a45c4145b.